bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOIntegrationTest.java

```diff
@@ -102,44 +116,78 @@ public void testEstimatedAndSplitForSmallTable() throws Exception {
   public void testEstimatedAndSplitForLargeTable() throws Exception {
```

Comment:
Would this help:

Assert.assertSame(source.getSampleRowKeys(), source.getSampleRowKeys())